frontend/src/model/beta/settings/ModelAccess.tsx

```diff
@@ -55,6 +56,7 @@ export default function ModelAccess({ model }: ModelAccessProps) {
   // TODO - add a request to update the model's collaborators field
   async function updateAccessList() {
     await patchModel(model.id, { collaborators: accessList })
+    setLoading(true)
```

Comment:
Loading never disabled!

Reply:
It's already been pointed out, but you'll need to `setLoading(false)` if `patchModel` errors. Unfortunately this request doesn't have any error handling at the moment (although it definitely should!), so you'll need to add that in also (the TODO is not for you to do in this ticket):
```
// You'll need a new state variable
const [patchModelErrorMessage, setPatchModelErrorMessage] = useState('')

async function updateAccessList() {
    setLoading(true)

    const response = await patchModel(model.id, { collaborators: accessList })

    if (!response.ok) {
      const error = await getErrorMessage(response)
      setLoading(false)  // Notice we have asynchronous logic on the line above, so we also want to wait for that to finish before setting the loading state back to false
      setPatchModelErrorMessage(error)
    }

    // TODO - Provide user feedback on success
}

// You'll also need to add the following (I've included the isUsersError logic so you know where to put it):
if (patchModelErrorMessage) {
  return <MessageAlert message={patchModelErrorMessage} severity='error' />
}

if (isUsersError) {
  return <MessageAlert message={isUsersError.info.message} severity='error' />
}
```